client/my-sites/plugins/plugins-browser-list/test/index.jsx

```diff
@@ -6,6 +6,10 @@ jest.mock( 'i18n-calypso', () => ( {
 	localize: ( c ) => c,
 	translate: ( s ) => s,
 } ) );
+jest.mock( 'react-redux', () => ( {
```

Comment:
@lupus2k Why is this needed? To dispatch the track event?

Reply:
Exactly, useDispatch comes from react-redux